lapce-app/src/proxy/remote.rs

```diff
@@ -116,20 +116,37 @@ pub fn start_remote(
         _ => format!("{remote_proxy_path}/lapce"),
     };
 
-    if !remote
+    let mut remote_version = remote
         .command_builder()
-        .args([&remote_proxy_file, "--version"])
+        .args(["lapce-proxy", "--version"])
```

Comment:
This will cause the Lapce managed proxies fail to break out quickly, which is the majority.

Reply:
I don't understand what you mean

Reply:
So for remotes that Lapce self installs, it won't be in PATH, which means this will always fail for them.

Reply:
Yes, that is expected

Reply:
I don't think that's a good idea, since most will be Lapce installed ones.

Reply:
I don't think it's that big of a deal since it will always fallback to script

Reply:
So I connect to my remote server every day, and I don't want it to fallback to script which is much faster.